main.py

```diff
@@ -184,17 +196,17 @@ def collision():
 # GAME LOOP
 while run:
 	# Draws the stuff to be displayed in window
-	bgX, bg_width, groundX, ground_width = redrawWindow(bgX, bg_width, groundX, ground_width)
+	bgX, bg_width, groundX, ground_width = draw_stuff_on_window(bgX, bg_width, groundX, ground_width)
 
 	# Gets the mouse pointer coordinates
 	(mx, my) = pygame.mouse.get_pos()
 
 	# Displaying the mouse pointer coordinates for reference
-	display_pointer(mx,my)
+	display_mouse_pointer_coordinates(mx,my)
 	   	
-	# To limit the movable region
+	# To limit the movable region of player

```

Comment:
Comments should be as minimal as possible and still express everything the programmer wants to convey.
For eg: 'limit players movable region'